### ISSUE TITLE
mod/pulse:sender capability

### ISSUE THREAD

**Issue by superfoght** [bdecentgmbh/moodle-mod_pulse#24]
Hi all,

Thanks for a nice plugin. I have a client that is interested in controlling who the sender of the notifications are. I found this in your documentation.

_Once the students to notify are identified, Pulse finds the correct sender user, who is the user with the "mod/pulse:sender" capability. When an instance has more than one sender, Pulse will use the first user as the sender. If the course is in group mode, Pulse will use the sender from the group._

However, that particular capability does not seem to be available? 

Thanks in advance for your answer :)

**Comment by taymossimc** [comment]
+1 for the ability to select the sender as part of the configuration of the pulse.

superfoght, it uses the first person (in time order) assigned "Teacher" role as the sender. I haven't found a way to manipulate the sender accept to take people out of the teacher role.

**Comment by superfoght** [comment]
> superfoght, it uses the first person (in time order) assigned "Teacher" role as the sender. I haven't found a way to manipulate the sender accept to take people out of the teacher role.

Yeah, but in the documentation it says that there is this capability:
_Once the students to notify are identified, Pulse finds the correct sender user, who is the user with the "mod/pulse:sender" capability. When an instance has more than one sender, Pulse will use the first user as the sender. If the course is in group mode, Pulse will use the sender from the group._

Since our client wants a different role to be the sender, I wanted to switch the capability between the roles. But I couldn't find that capability anymore. So eithere there is a bug here or the documentation needs to be updated :)

**Comment by stefanscholz** [comment]
Hi @superfoght,

the capability does exist, the documentation is correct:

![Bildschirmfoto 2023-10-06 um 13 21 36](https://github.com/bdecentgmbh/moodle-mod_pulse/assets/3414705/42f91367-af9b-45d5-a2a8-bb29b8b647b2)

Not sure why you don't see it. Which version of Pulse have you reviewed?

Side note: in Pulse 2.0, we are introducing a completely new way to manage notifications as part of our course automation framework. There, you will be able to pick the sender per notification. We hope this makes it easier for you!

Hope this helps!
Stefan

@taymossimc fyi

**Comment by superfoght** [comment]
That's really odd we are using version 1.2

![image](https://github.com/bdecentgmbh/moodle-mod_pulse/assets/2778015/fd14b65a-6c1f-4420-8121-60870bcca11c)

But I only see these capabilities being available? 

![image](https://github.com/bdecentgmbh/moodle-mod_pulse/assets/2778015/11aa3c52-be6c-4e78-838c-3190ee607e29)

Does it make a difference that we have it installed on Moodle Workplace?
